TL improvement (#524)
07th-mod/higurashi-console-arcs: Update/omot_003.txt

//r思ったより時間が経っていないことに、軽く失望する。今まで、時間の流れが速すぎると焦ることはあっても、遅すぎるとうんざりすることなんて、一度もなかったのだけど。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "思ったより時間が経っていないことに、軽く失望する。今まで、時間の流れが速すぎると焦ることはあっても、遅すぎるとうんざりすることなんて、一度もなかったのだけど。",

> 			NULL, "I was a bit disappointed by how little time had passed. Before now, while sometimes I had get flustered by the rapid flow of time, I had never felt weary of the sluggish flow, though.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I was a bit disappointed by how little time had passed. Before now, there were times I got flustered by the rapid flow of time, but I had never felt weary of the sluggish flow.", Line_Normal);
```

//rでも、それだけ。答えてくれるわけじゃない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、それだけ。答えてくれるわけじゃない。",

> 			NULL, "But that was all. The sheet cover had no answer for that.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But that was all. It had no answer for me.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600132", 256, TRUE);
	OutputLine(NULL, "「夏バテにしては時期はずれですね…。",

> 			NULL, "\"Heat fatigue is out of season, though...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"This isn't the season for heat fatigue, though...", Line_WaitForInput);
```

			NULL, "\"Please don't worry about me.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500055", 256, TRUE);
	OutputLine(NULL, "来てくれないと沙都子の個性的なご飯になってしまうので、ちょっとだけ大変なのです」",

> 			NULL, " If you don't come over, I'll be able to eat Satoko's unique dishes. It's only a little awful.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " If you don't come over, I'll be able to eat Satoko's unique meals. It's only a little awful.\"", Line_Normal);
```

//r不愉快なのは、見透かされているように感じるからだ。『オヤシロさま』の名前を出しても、今はもう、私のそばにはいない――そのことを、指摘されているようで…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "不愉快なのは、見透かされているように感じるからだ。『オヤシロさま』の名前を出しても、今はもう、私のそばにはいない——そのことを、指摘されているようで…。",

> 			NULL, "The reason why I felt so unpleasant was that I felt like she was seeing through me. I had mentioned the name of \"Oyashiro-sama,\" the one who was no longer by my side... She looked like she was showing me that point...", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was unpleasant because it felt like she was seeing through me. I had mentioned the name of \"Oyashiro-sama,\" the one who was no longer by my side... And it seemed like she was emphasizing that point...", Line_Normal);
```

//r詩音は、雛見沢から隔離された時間が長い。そしてそれだけ、これまでの時間でも崇りについては比較的現実的で、そして若干懐疑的ですらあった…ただ一点を除いて、だが。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "詩音は、雛見沢から隔離された時間が長い。そしてそれだけ、これまでの時間でも崇りについては比較的現実的で、そして若干懐疑的ですらあった…ただ一点を除いて、だが。",

> 			NULL, "Shion had been isolated from Hinamizawa for a long time. That's why, up until now, she had realistic views on the curse, even a bit of skepticism... with one particular exception, though.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Shion had been isolated from Hinamizawa for a long time. That's why up until now, she had realistic views on the curse, even a bit of skepticism... with one particular exception, though.", Line_Normal);
```

			NULL, "\"Meep... I'm starting to feel a little better after getting some rest.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500059", 256, TRUE);
	OutputLine(NULL, "ご飯を食べたら頑張って、お昼の授業は出てみます」",

> 			NULL, " I'll eat lunch and manage to head back to afterwards.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I'll eat lunch and do my best to go to class afterwards.\"", Line_Normal);
```

//rしまった、と思ったが、その前に詩音は肩をすくめて、おどけた表情で笑う。笑うなら笑え、とでも言いたげだったが、意外にもその目には強い意志が宿っていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しまった、と思ったが、その前に詩音は肩をすくめて、おどけた表情で笑う。笑うなら笑え、とでも言いたげだったが、意外にもその目には強い意志が宿っていた。",

> 			NULL, "I thought it was a mistake to mention his name, but Shion quickly shrugged her shoulders and flashed a playful smile. It was as if she was saying \"go ahead and laugh,\" but unexpectedly enough, she had a strong will in her eyes.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I thought it was a mistake to mention his name, but Shion quickly shrugged her shoulders and flashed a playful smile. It was like she was saying \"go ahead and laugh,\" but surprisingly enough, she had a strong will in her eyes.", Line_Normal);
```

//r知っている。そして私がたどってきたある時間では、それが高じて園崎家、そして沙都子への殺意と変わっていった…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "知っている。そして私がたどってきたある時間では、それが高じて園崎家、そして沙都子への殺意と変わっていった…。",

> 			NULL, "I already know about all that. In one of the worlds I had reached, it warped into intense murderous rage against the Sonozaki family, and against Satoko...", Line_Normal);

Reviewer: ```suggestion
			NULL, "I already know about all that. In one of the worlds I followed, it warped into intense murderous rage against the Sonozaki family, and against Satoko...", Line_Normal);
```

			NULL, "\"Anyway, after that baseball game a little while back...", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600166", 256, TRUE);
	OutputLine(NULL, "圭ちゃんがきっかけで話をしたんです」",
			NULL, " I had a chance to chat with Satoko thanks to Kei-chan.\"", Line_Normal);
	ClearMessage();

//詩音rvS14/06/210600167.「で、ちょっと意地悪な気持ちで質問してみました。kvS14/06/210600168.『悟史くんが帰ってきたら、真っ先に何をしてあげます？』って。kvS14/06/210600169.…その時にあの子、なんて答えたと思います？」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600167", 256, TRUE);
	OutputLine(NULL, "「で、ちょっと意地悪な気持ちで質問してみました。",

> 			NULL, "\"So, I was lightly teasing her while asking a question.", Line_WaitForInput);

Author: Mistake. "So, I tried lightly teasing her while asking a question." is accurate.

Reviewer: ```suggestion
			NULL, "\"So, I tried lightly teasing her while asking a question.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500079", 256, TRUE);
	OutputLine(NULL, "「…ボクも沙都子も賛成しますですよ」",

> 			NULL, "\"...Satoko and I agree with you too.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...Satoko and I agree.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 4, "ps3/s14/04/210400099", 256, TRUE);
	OutputLine(NULL, "「あの二人だとしたら、思いっきり参加条件を厳しくしてやりたいところですわ～！」",

> 			NULL, "\"I'll raise the bar extremely high for those two if they apply for it~!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"If it's those two, I'll raise the bar for application extremely high~!\"", Line_Normal);
```
